examples/profilesandbox/test_sandboxanchorexample.go

```diff
@@ -0,0 +1,59 @@
+package profilesandbox
+
+import (
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/getyoti/yoti-go-sdk/v2"
+	"github.com/getyoti/yoti-go-sdk/v2/cryptoutil"
+	"github.com/getyoti/yoti-go-sdk/v2/file"
+	"github.com/getyoti/yoti-go-sdk/v2/profile/sandbox"
+
+	_ "github.com/joho/godotenv/autoload"
+	"gotest.tools/v3/assert"
+)
+
+func TestAnchorExample(t *testing.T) {
+	sandboxClientSdkId := os.Getenv("SANDBOX_CLIENT_SDK_ID")
+	pemFileBytes, err := file.ReadFile(os.Getenv("YOTI_KEY_FILE_PATH"))
+	assert.NilError(t, err)
+
+	privateKey, err := cryptoutil.ParseRSAKey(pemFileBytes)
+	assert.NilError(t, err)
+
+	sandboxClient := sandbox.Client{
+		ClientSdkID: sandboxClientSdkId,
+		Key:         privateKey,
+	}
+
+	sourceAnchor := sandbox.SourceAnchor("NFC", time.Now().UTC(), "PASSPORT")
+	verifierAnchor := sandbox.VerifierAnchor("", time.Now().UTC(), "YOTI_ADMIN")
+
+	familyNameAttributeWithAnchors := sandbox.Attribute{}.
```

Comment:
Is is worth having an example providing anchors to pre-defined attributes like `WithGivenNames`, as it's likely integrators will start with these?

Reply:
Good point, updated in 8f75d50